langfuse/llama_index/llama_index.py

```diff
@@ -92,13 +106,49 @@ def __init__(
         self.event_map: Dict[str, List[CallbackEvent]] = defaultdict(list)
         self._llama_index_trace_name: Optional[str] = None
         self._token_counter = TokenCounter(tokenizer)
+        self.tags = tags
 
         # For stream-chat, the last LLM end_event arrives after the trace has ended
         # Keep track of these orphans to upsert them with the correct trace_id after the trace has ended
         self._orphaned_LLM_generations: Dict[
             str, Tuple[StatefulGenerationClient, StatefulTraceClient]
         ] = {}
 
+    def set_trace_metadata(
```

Comment:
Please ensure that the context is cleared after llamaindex invocation so that no succeeding llamaindex usage gets the same contexts.